assets/js/vue/FileManager.vue

```diff
@@ -25,6 +15,46 @@
                 </p>
             </template>
         </DxPopup>
+        <DxPopup
+            title="Uploading Files"
+            :visible.sync="loadingVisible"
+            :close-on-outside-click="false"
+            :show-title="false"
+            position="center"
+            :showCloseButton="false"
+            :width="300"
+            :height="250"
+            :drag-enabled="false"
+        >
+            <template>
+                <center>
+                <p>
+                    {{ uploadMessage }}
+                </p>
+                <p>
+                    {{ bytesMessage }}
+                </p>
+                </center>
+                <DxProgressBar
+                  id="progress-bar-status"
+                  :min="0"
+                  :max="totalFileSize"
+                  :value="doneFileSize"
+                  width="90%"
+                />
+                <center>
+                <DxButton
+                    text="Cancel Upload!"
+                    type="danger"
+                    styling-mode="contained"
+                    @click="stopProcess"
+                 />
+                 <p>
+                    If the filesize is large? Click Cancel!
```

Comment:
This message is more of a placeholder. It should appear maybe when large file size is detected?

Reply:
I think the generic message is good, we can ask the PO about what they think